metabase/metabase
# Mantine menu

Closes https://github.com/metabase/metabase/issues/32273

Changes:
- Exports `Menu` and adds style overrides
- Exports `Button` for testing, yet to be styled in a separate PR
- Changes `spacing`, `radius`, `shadows` values to what we use in the project, and updates overrides 
- Moves component overrides to the component folders

How to verify:
- Run `yarn storybook`
- Open http://localhost:6006/?path=/story/layovers-menu--with-label-and-divider
- Make sure it works according to the figma file https://www.figma.com/file/MZhwfwmOaa8HeCBBUCeq7R/Menu?type=design&node-id=5-70&mode=design&t=1cSNiSttrRRUaZGV-0

<img width="992" alt="Screenshot 2023-08-11 at 16 09 48" src="https://github.com/metabase/metabase/assets/8542534/a95e81c7-4183-4dc7-b5a1-ea92aedabe49">
